bin/kano-signal

```diff
@@ -43,6 +46,31 @@ except:
     xml_filename = ""
 
 
+def set_focus(app, verbose=False):
+    '''
+    Sets user input focus to the app name.
+    '''
+    app_titles_dict = {
+        'make-minecraft': 'Make Minecraft',
+        'make-pong': 'Make Pong',
+        'sonicpi': 'Sonic Pi',
+        'make-light': 'Make Light'
+    }
+
+    if app in app_titles_dict:
+        # switch user focus back to the game app.
+        cmd = 'wmctrl -a "{}"'.format(app_titles_dict[app])
+        if verbose: print cmd
+        rc = os.system(cmd)
+        if rc != 0:
+            from kano.logging import logger
+            logger.error("error raising window {}".format(app_titles_dict[app]))
```

Comment:
This would probably be better with an `itervalues`:

```python
if app in app_titles_dict.itervalues():
    # switch user focus back to the game app.
    cmd = 'wmctrl -a "{}"'.format(app)
    if verbose: print cmd
    rc = os.system(cmd)
    if rc != 0:
        from kano.logging import logger
        logger.error("error raising window {}".format(app))
```

Also, it seems like this dict is only created to run through the apps list - would it not be better to just use a list?